.github/workflows/build.functions_tests.sh

```diff
@@ -53,7 +58,8 @@ function assert_get_hz_dist_tar_gz {
   local distribution=$2
   local expected_url=$3
   local actual_url=$(get_hz_dist_tar_gz "$hz_version" "$distribution")
-  assert_eq "$expected_url" "$actual_url" "Expected URL for version \"$hz_version\", distribution \"$distribution\"" || TESTS_RESULT=$?
+  local msg="Expected URL for version \"$hz_version\", distribution \"$distribution\""
```

Comment:
Could you add `$expected_url` to `$msg` to see what exactly we expect, currently we see only inputs:

![image](https://github.com/user-attachments/assets/95e849b6-7f80-4501-a5b4-07dcf2681404)

Reply:
70ad43a96fc5014e0f3bdd49f1b495a2dd8b6484